packages/nimble-components/src/wafer-map/template.ts

```diff
@@ -1,4 +1,36 @@
-import { html } from '@microsoft/fast-element';
+import { html, repeat } from '@microsoft/fast-element';
 import type { WaferMap } from '.';
+import type { WaferMapDie } from './types';
 
-export const template = html<WaferMap>``;
+const notReadyTemplate = html<WaferMap>` <div>Loading....</div> `;
+
+const waferTemplate = html<WaferMap>`
```

Comment:
We should use the terminology `waferMap` consistently

```suggestion
const waferMapTemplate = html<WaferMap>`
```

Reply:
Applied